### DISCUSSION TITLE
Upgrading to the latest version.

### DISCUSSION THREAD

**Discussion by Sagzster** [NginxProxyManager/nginx-proxy-manager#1654] [Q&A]
Sorry for the dumb question...I've been using and enjoying 2.9.4 and recently went and started upgrading vulnerable software on my Proxmox server and don't know how to upgrade docker-compose containers.  I couldn't figure this out and its my only docker-compose service.  Any help would be greatly appreciated!

**Comment by chaptergy** [comment]
It's the same as for almost all dockerized applications, have a look at [this documentation page](https://nginxproxymanager.com/upgrading/). Note that this is the procedure if you use the docker image tag `:latest`. If you use a specific version number as the docker tag, e.g. `:2.9.4`, just change the tag to (currently) `:2.9.12` and restart the container.

**Comment by Sagzster** [comment]
Sorry I'm just getting back to this.  I did the:
docker-compose pull
which gave me an error which I tracked down to this post:
https://github.com/jc21/docker-mariadb-aria/issues/8
I made the changes to the docker-compose.yml and created the my.cnf file
but now I cannot log into NPM.
Username and password just sits there after I hit "Sign in"

On a positive note the version is showing v.2.9.12

Anyone have an idea what to do next?

**Reply by Sagzster** [reply]
Does anyone have a suggestion as to where to look to see why I can't log in?
